spec/support/features/session_helper.rb

```diff
@@ -210,6 +211,11 @@ def sign_in_and_2fa_user(user = user_with_2fa)
       user
     end
 
+    def sign_in_and_2fa_user_with_remember_device(user = user_with_2fa)
+      sign_in_with_warden(user, auth_method: 'remember_device')
+      user
+    end
```

Comment:
I find it hard to keep track of all our various sign in helpers. since this is only used once, WDYT of inlining it to the spec?

```ruby
user = user_with_2fa
sign_in_with_warden(user, auth_method: 'remember_device')
```

Reply:
yep, makes sense

Reply:
removed